src/course/en/components.json

```diff
@@ -26,10 +27,15 @@
         "body": "",
         "instruction": "",
         "_graphic": {
-            "large": "course/en/images/single_width.jpg",
-            "small": "course/en/images/single_width.jpg",
+            "large": "course/en/images/single-width.png",
+            "small": "course/en/images/single-width.png",

```

Comment:
it would be great if we could actually have an example of using a different image for small screens... but totally a nice-to-have so if you don't have time don't worry